### ISSUE TITLE
allow selecting job_templates by feature when creating job_invocations

### ISSUE THREAD

**Issue by evgeni** [theforeman/foreman-ansible-modules#1234]
##### SUMMARY
`job_invocation` currently expects the user to select the template by providing a `job_template` name, however the API also allows to select them by "feature"

apidoc says:
job_invocation[feature]                 optional         , nil allowed | Remote execution feature label that should be triggered, job template assigned to this feature will be used

should be marked as "required_one_of(job_tempalte, feature)" and I think also "mutually_exclusive(job_template, feature)"

##### ISSUE TYPE
 - Feature Idea

**Comment by evgeni** [comment]
props to @jturel for teaching me this!

**Comment by jturel** [comment]
> should be marked as "required_one_of(job_tempalte, feature)" and I think also "mutually_exclusive(job_template, feature)"
> 

Correct, the API will error if both are specified and one of them should be included with each create request
